extractor/src/main/java/org/schabi/newpipe/extractor/stream/AudioStream.java

```diff
@@ -506,21 +500,9 @@ public Locale getAudioLocale() {
         return audioLocale;
     }
 
-    /**
-     * Returns whether this stream is a descriptive audio.
-     *
-     * <p>
-     * A descriptive audio is an audio in which descriptions of visual elements of a video are
-     * added in the original audio, with the goal to make a video more accessible to blind and
-     * visually impaired people.
-     * </p>
-     *
-     * @return {@code true} this audio stream is a descriptive audio, {@code false} otherwise
-     * @see <a href="https://en.wikipedia.org/wiki/Audio_description">
-     *     https://en.wikipedia.org/wiki/Audio_description</a>
-     */
-    public boolean isDescriptive() {
-        return isDescriptive;
+    @Nullable
+    public AudioTrackType getAudioTrackType() {
+        return audioTrackType;
     }
```

Comment:
Please add documentation for this getter, this one will be likely the one which will be used the most by clients.